main.css

```diff
@@ -0,0 +1,71 @@
+h1 {
+    font-family: Georgia, "Times New Roman", Times, serif;
+    font-size: 2em;
+    text-align: center;
+    
+}
+h4 {
+    padding-left: .5em;
+    font-size: 1.2em;
+    font-weight: bold;
+}
+h3 {
+    padding-left: 1.5em;
+}
+footer {
+    font-size: .75em;
+}
+header {
+    border: 1px dotted gray;
+}
+
+.cards {
+    border: 1px solid lightblue;
+    width: 30%;
+    display: inline-block;
+}
+
+   
+.cards1 {
+    border: 1px solid lightblue;
+    width: 31%;
+    float: left;
+    /* padding: 10px; */
+    margin: 10px;
+}
```

Comment:
For Requirement 2 you needed to keep the width of each card to 30%, not 31%. You had it right the first  time with your 'card' class instead of 'card1'